components/table/SearchFilterSort.tsx

```diff
@@ -104,22 +140,26 @@ export default function SearchFilterSort() {
 
           <span className="md:inline hidden">Filters</span>
         </Disclosure.Button>
-        <button
-          type="button"
-          className="inline-flex items-center gap-x-2 rounded-md bg-white px-3 py-1.5 text-sm font-semibold text-darkergrey shadow-sm ring-1 ring-inset ring-gray-300 hover:bg-gray-50"
-        >
-          <Image
-            alt="Export"
-            src={Export}
-            width={16}
-            height={12.5}
-            style={{
-              maxWidth: '100%',
-              height: 'auto',
-            }}
-          />
-          <span className="md:inline hidden">Export</span>
-        </button>
+
+        {pathname.split('/')[2] === 'products' ? (
```

Comment:
This can be fragile to depend on the pathname. Let's takes a prop to display the download button or not